docs/build/Build-for-OpenWrt.md

```diff
@@ -0,0 +1,99 @@
+# IoT.js for OpenWrt build guide
+
+This document describes the steps required to compile the IoT.js
+for OpenWrt. For target device the TP-Link WR1043ND v1.x router is
+used. Please be advised, that if you have a different one minor
+modifications to this document could be required.
+
+
+IMPORTANT!
+
+As the TP-Link WR1043ND is a mips based device and mips is a big-endian
+architecture. A JerryScipt snapshot which was built on an little-endian
+system will not work correctly. Thus the IoT.js must be
+built with disabled snaphost mode.
+
+## OpenWrt notes
+
+In 2018 ~January the OpenWrt and LEDE project merged into one
```

Comment:
Suggestion:
Since January 2018, the OpenWrt and LEDE project have merged into one .......